src/sentry/incidents/endpoints/project_alert_rule_index.py

```diff
@@ -24,6 +24,7 @@ def get(self, request: Request, project) -> Response:
         """
         Fetches alert rules and legacy rules for a project
         """
+        expand = request.GET.getlist("expand", [])  # type: ignore
```

Comment:
please fix this before merging -- it's because type comments are not supported by flake8 (python 2 only)